container-engine-lib/lib/user_support_constants/user_support_constants.go

```diff
@@ -30,8 +30,10 @@ const (
 	KurtosisOnBoardCalendlyUrl        = "https://calendly.com/d/zgt-f2c-66p/kurtosis-onboarding"
 	FeedbackEmail                     = "feedback@" + OldDomain
 	FeedbackEmailLink                 = "mailto:" + FeedbackEmail
-	KurtosisTechTwitterProfileLink    = "https://twitter.com/KurtosisTech"
-	KurtosisCloudLink                 = "https://cloud." + Domain
+	// TODO: Address issue: https://github.com/kurtosis-tech/kurtosis/issues/1968 and add this back to urlsToValidateInTest
```

Comment:
you can remove this todo, it was fixed here: https://github.com/kurtosis-tech/kurtosis/pull/1974/files

Reply:
awesome!